samples/micrometer-samples-boot2/build.gradle

```diff
@@ -14,12 +14,15 @@ dependencyManagement {
 }
 
 dependencies {
+    implementation project(":micrometer-core")
     ['atlas', 'azure-monitor', 'prometheus', 'datadog', 'elastic', 'ganglia', 'graphite', 'jmx', 'influx', 'statsd', 'new-relic', 'cloudwatch', 'cloudwatch2', 'signalfx', 'wavefront', 'elastic', 'dynatrace', 'humio', 'appoptics', 'stackdriver'].each { sys ->
         implementation project(":micrometer-registry-$sys")
     }
 
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation('org.springframework.boot:spring-boot-starter-actuator') {
+        exclude group: 'io.micrometer', module: 'micrometer-core'
```

Comment:
Do we need to explicitly exclude this here or does the direct declaration of `implementation project(":micrometer-core")` override the transitive dependency? Just curious.

Reply:
@shakuzen I thought the same thing, but it seems to be necessary based on the build results [before](https://circleci.com/gh/izeye/micrometer/680) and [after](https://circleci.com/gh/izeye/micrometer/682) the exclusion although I didn't look into it.